### PR TITLE
tests: followup on #2656

### PR DESCRIPTION
**Pull request by guits**
34f70428 has introduced a fix using `command` module while this could have been achieved by using `lvol` module.

Signed-off-by: Guillaume Abrioux <gabrioux@redhat.com>

### CODE REVIEW

File: tests/functional/setup.yml [hunk 1]
```diff
@@ -32,9 +32,9 @@
...
 
     - name: resize logical volume for root partition to fill remaining free space
-      command: lvresize -l +100%FREE /dev/atomicos/root
-      when: is_atomic
-
-    - name: increasing the size of the fs for root partition
-      command: xfs_growfs /
+      lvol:
+        lv: root
+        vg: atomicos
```
Comment (ErwanAliasr1) [inline tests/functional/setup.yml]: looks like a mix of space/tabs isn't it ?

Comment (guits) [reply tests/functional/setup.yml]: L37 was an empty line while L36 gets two more spaces.

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-docker_collocation

**Comment by guits** [comment]
jenkins test luminous-ansible2.4-docker_cluster_collocation

**Comment by guits** [comment]
We must wait ceph-ansible testing against ansible 2.5 for this patch since `resizefs` parameter has been added in ansible 2.5

**Comment by guits** [comment]
jenkins test pipeline

**Pull request merged**